src/main/scala/org/scalajs/tools/tsimporter/sc/Definitions.scala

```diff
@@ -271,6 +273,8 @@ object TypeRef {
 
   val ScalaAny = TypeRef(scala dot Name("Any"))
 
+  val Wildcard = TypeRef(Name("_",false))
```

Comment:
Oh, I'm sorry! I thought there was already a mechanism for this, but I mixed up with the type parameters of a class/interface definition :-s

Then this doesn't make any sense. I'll have to think more seriously about how to introduce upper bounds and wildcards in the model. But in the meantime, could you please revert those changes? i.e., not changing `Definitions.scala`, and using `js.Array[Any]` as you did before.

Sorry, I thought the thing was simpler.